app/assets/javascripts/angular/modules/news-title-parse.js

```diff
@@ -0,0 +1,9 @@
+angular.module('parseNews', []).filter('parseTitle', function () {
```

Comment:
We generate it [here](https://github.com/MegaGameSociety/WatchTheSkiesDashboard/blob/develop/app/models/tweet.rb#L23) when converting a tweet into a news article.  Totally fine to remove it.